api/src/api.ts

```diff
@@ -112,6 +113,33 @@ export const plugin = {
       },
     });
 
+    server.route({
+      method: "GET",
+      path: "paper/{s2Id}",
+      handler: async (request) => {
+        const s2Id = request.params.s2Id;
+        let resp = null;
+        try {
+          resp = await s2Api.getPaperUncached(s2Id, config.s2.apiKey);
+        } catch (error) {
```

Comment:
You can return within the try:
```
try { 
  const resp = await s2Api.getPaperUncached(s2Id, config.s2.apiKey);
  return resp?.data;
} catch (error) {
```

Unless the style of this project is putting returns at the bottom, I guess.